organizers/2020-01-15.html

```diff
@@ -0,0 +1,144 @@
+---
+layout: organizers
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>January 15, 2020</h2>
+    <img data-src="/img/django-logo-white-transparent.svg" style="width:40%; background: none" alt="Django logo" class="plain">
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other </h2>
+    <p>The mission of The London Django Meetup Group is to provide a fun, welcoming and professional environment for a diverse group of people.</p>
+    <p>
+        Code of conduct: <a href="https://www.djangolondon.com/code-of-conduct/">djangolondon.com/code-of-conduct</a>
+    <p>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+      <li><strong>Festicket</strong></li>
+      <li><strong>Pollen (formerly Verve)</strong></li>
+      <li><strong>Time Out</strong></li>
+      <li><strong>Yelp</strong></li>
```

Comment:
Not mentioning swag donors here (even jetbrains) - they're all on my quiz slides